be/src/runtime/data_stream_recvr.cpp

```diff
@@ -237,17 +241,18 @@ Status DataStreamRecvr::add_chunks(const PTransmitChunkParams& request, ::google
     MemTracker* prev_tracker = tls_thread_status.set_mem_tracker(_instance_mem_tracker.get());
     DeferOp op([&] { tls_thread_status.set_mem_tracker(prev_tracker); });
 
-    SCOPED_TIMER(_process_total_timer);
-    SCOPED_TIMER(_sender_total_timer);
-    COUNTER_UPDATE(_request_received_counter, 1);
+    auto& metrics = get_metrics_round_robin();
+    SCOPED_TIMER(metrics.process_total_timer);
+    SCOPED_TIMER(metrics.sender_total_timer);
```

Comment:
can we remove one of these two timers? it seems that these two values are always same

Reply:
yes, I'll remove sender_total_timer.